README.md

```diff
@@ -49,25 +49,49 @@ Note: some version issues can create poor FIDs. Always check your build
 environment first with the `evaluate` method. With the best FFHQ score you
 should always get under an FID < 2.10 (hopefully closer to 2.05). 
 
```

Comment:
```suggestion

Note: 
[NATTEN can be sped up by using pre-built wheels directly.](https://shi-labs.com/natten/)
```
I think this might be removed in error.